apps/portals/src/configurations/elportal/resources.ts

```diff
@@ -8,7 +8,7 @@ export const studiesSql = 'SELECT * FROM syn51209885 ORDER BY isFeatured DESC'
 export const speciesSql = 'SELECT * FROM syn51449110 ORDER BY ItemOrder'
 export const computationalSql = 'SELECT * FROM syn51469335'
 export const upsetPlotSql = 'SELECT files, assay FROM syn51581771'
-export const cohortBuilderSql = 'SELECT * FROM syn52239906'
+export const cohortBuilderSql = 'SELECT * FROM syn52234652'
```

Comment:
syn52239906 deleted.  Looks like syn52234652 should be right, but have yet to test